### ISSUE TITLE
Add highlighting functionality

### ISSUE THREAD

**Issue by ArashBayatDev** [khramts/assocplots#5]
Hi,
    Would it be possible to add an option to show some specific variant sites with a different colors.
    For example the function accept an array of (chr,pos,p-value,color) and display this list of variant on top the Manhattan plot.

**Comment by khramts** [comment]
Yes, I will look into making such modifications in the code, however I would be able to work on this only in 1-2 weeks.

**Comment by ArashBayatDev** [comment]
Thanks.

==============================
Arash Bayat
Researcher
Transformational Bioinformatics
Health and Biosecurity Business Unit
CSIRO
E arash.bayat@csiro.au<mailto:arash.bayat@csiro.au> T +61 2 9325 3090
Postal address: PO Box 52, North Ryde, NSW 1670, Australia | Street address: Level 4 Building 53, Riverside Life Sciences Centre (RLSC), Riverside Corporate Park, 11 Julius Avenue, North Ryde NSW 2113
https://aehrc.com/biomedical-informatics/transformational-bioinformatics/
http://bioinformatics.csiro.au<http://bioinformatics.csiro.au/>

CSIRO acknowledges the Traditional Owners of the lands that we live and work on across Australia and pays its respect to Elders past and present.
PLEASE NOTE - The information contained in this email may be confidential or privileged. Any unauthorised use or disclosure is prohibited. If you have received this email in error, please delete it immediately and notify the sender by return email. Thank you. To the extent permitted by law, CSIRO does not represent, warrant and/or guarantee that the integrity of this communication has been maintained or that the communication is free of errors, virus, interception or interference.
Please consider the environment before printing this email.

**Comment by Bochao1** [comment]
Hi, 
I am wondering is highlighting function is available now for assocplot? What is the arguments? 
Very looking forward to use that.

**Comment by ArashBayatDev** [comment]
Good to know that you have implemented Highlighting function.
Will test it in case I need it in further analysis

**Comment by khramts** [comment]
Hello everyone,
I have not worked on this function and currently I am short on time to work
on it. If this function becomes available, I will post a comment.
Regards,
Katya
